components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -291,6 +293,10 @@ class RewardsServiceImpl : public RewardsService,
 
   void GetExternalWallet(GetExternalWalletCallback callback) override;
 
+  std::string GetExternalWalletType() const;
+
+  const std::vector<std::string> GetExternalWalletProviders() const override;
```

Comment:
The `const` here prevents move semantics (`const` rvalues can't bind to rvalue reference to non-`const`s).